docs/UserGuide.md

```diff
@@ -360,6 +360,25 @@ Format: `delete_session n/STUDENT_NAME i/SESSION_INDEX`
 Example:
 * `delete_session n/John Lee i/1` deletes John Lee's **first** session
 
+#### Deleting a recurring tuition session: `delete_rec_session`
```

Comment:
Looks good! Do remember to include this in the TOC and the command summary as well!

Reply:
Oh crap forgot about that thank you!